### PR TITLE
[IMP] finance/acquirers: alipay and ogone are now deprecated

### PR DESCRIPTION
**Pull request by Feyensv**
See also:
- https://github.com/odoo/odoo/pull/99025
- https://github.com/odoo/upgrade/pull/3848

### CODE REVIEW

File: content/applications/finance/payment_acquirers/alipay.rst [hunk 1]
```diff
@@ -5,6 +5,10 @@ Alipay
 `Alipay <https://www.alipay.com/>`_ is an online payments platform established in China by Alibaba
 Group.
 
+.. warning::
+
+    Alipay is now deprecated.  As of version 16.0, the module won't be maintained anymore.
```
Comment (Feyensv) [inline content/applications/finance/payment_acquirers/alipay.rst:10]:
```suggestion
    The provider Alipay is deprecated.  It is recommended to use :doc:`asiapay` instead.
```

**Review by jcs-odoo — COMMENTED** [review]
@Feyensv 
If they're deprecated for 16.0+, may I suggest we remove these pages, and add redirections for these pages to a note or section in the general provider doc?
What's the point of keeping pages that are not valid anymore?

**Comment by Feyensv** [comment]
We're not removing the feature, only making it not available for new instances anymore.
So the doc is still useful for the clients having the app already installed.
We simply do not want to promote them anymore ;).

**Comment by jcs-odoo** [comment]
> We're not removing the feature, only making it not available for new instances anymore. So the doc is still useful for the clients having the app already installed. We simply do not want to promote them anymore ;).

so these docs would be useful only for databases created prior a **date** and not really a version isn't it?
Does this mean we'd have to keep these docs forever, for all versions, though?

**Comment by Feyensv** [comment]
Nope, we will probably remove the feature fully in 17 (and then from the doc as well).
It's kinda a 'tampon' period where we encourage people to migrate from the old provider(s) to the new one(s), and then we remove the feature, but still not forcing the change directly without prior notice ;).

**Review by AntoineVDV — APPROVED** [review]

File: content/applications/finance/payment_acquirers/alipay.rst [hunk 2]
```diff
@@ -5,6 +5,10 @@ Alipay
 `Alipay <https://www.alipay.com/>`_ is an online payments platform established in China by Alibaba
 Group.
 
+.. warning::
+
+    The provider Alipay is deprecated.  It is recommended to use :doc:`asiapay` instead.
```
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/alipay.rst:10]:
```suggestion
.. warning::
   The provider Alipay is deprecated. It is recommended to use :doc:`asiapay` instead.
```

File: content/applications/finance/payment_acquirers/ogone.rst [hunk 3]
```diff
@@ -9,6 +9,10 @@ company that provides the technology involved in secure electronic transactions.
    - :ref:`payment_acquirers/add_new`
    - `Ogone's documentation <https://epayments-support.ingenico.com/get-started/>`_.
 
+.. warning::
+
+    The provider Ogone is deprecated.  It is recommended to use :doc:`stripe` instead.
```
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/ogone.rst:14]:
```suggestion
.. warning::
   The provider Ogone is deprecated. It is recommended to use :doc:`stripe` instead.
```

**Review by jcs-odoo — COMMENTED** [review]
I would understand if you hated me but I saw two spaces between `deprecated.` and `It is`.
🤣

(--> Je sors)

File: content/applications/finance/payment_acquirers/alipay.rst (hunk 2 above)
Comment (AntoineVDV) [reply content/applications/finance/payment_acquirers/alipay.rst:10]: That's actually a correct form of writing according to I-don't-remember-which-university-rulebook :p

**Review by jcs-odoo — APPROVED** [review]

**Comment by AntoineVDV** [comment]
Closing this one temporarily to allow the merge without conflict with https://github.com/odoo/documentation/pull/2392. It'll need to be reopened and rebased later.

**Pull request closed without merging**